docs/extras/code_samples/custom_v1.txt

```diff
@@ -1,20 +1,22 @@
-from mindee import Client, product
+from mindee import Client
+from mindee.parsing.common import PredictResponse
+from mindee.product import CustomV1
 
-# Init a new client and add your custom endpoint (document)
-mindee_client = Client(api_key="my-api-key").add_endpoint(
-    account_name="my-account",
-    endpoint_name="my-endpoint",
-)
+# Init a new client
+mindee_client = Client(api_key="my-api-key")
```

Comment:
Can't change that because of CI & testing, sadly :/

Reply:
@thentgesMindee if the API key is not specified when constructing the client object, it will automatically look for the `MINDEE_API_KEY` envvar.

https://developers.mindee.com/docs/python-getting-started#set-the-api-key-in-the-environment